### ISSUE TITLE
here's a tip

### ISSUE THREAD

**Issue by DeceptedPar** [ExtendedCentral/FNF-FANMADE-Chromatic-Scales#1]
if you use the samples with .wav the regions stay and it means people wont have to manually do them, if people were to manually do them, it'd take forever

**Comment by g3n-zu** [comment]
Oh my goodness! Thank you for this tip! I will update all chromatics to be .wav format for now on.

**Comment by DeceptedPar** [comment]
your welcome, I love what you've done so far though!

**Comment by g3n-zu** [comment]
Thank you!

**Comment by DeceptedPar** [comment]
if you can with the chromatics, always try to have an extended version cause people could make soundfonts with them

**Comment by DeceptedPar** [comment]
and then that'd help people making songs like the 
>But every other turn it's another character singing it

**Comment by g3n-zu** [comment]
True, True. Im gonna try and update old Chromatics with longer notes.

**Comment by g3n-zu** [comment]
Also, Do you happen to have a discord? I would Like to contact you for this new knowledge.

**Comment by DeceptedPar** [comment]
nope, we can always chat here

**Comment by g3n-zu** [comment]
I see, I see. I'm having issues with it auto-slicing with .wav. It's not auto slicing hehe. Maybe i'm doing it wrong?

**Comment by DeceptedPar** [comment]
I'm not to sure, if you press the icon for the region you can create a region and then move it around from there to where you want

**Comment by DeceptedPar** [comment]
![image](https://user-images.githubusercontent.com/64243677/131172442-b43c6c38-e135-4d01-ad3f-7e46873e3326.png)

**Comment by DeceptedPar** [comment]
you can also make loops, never tested how that works

**Comment by g3n-zu** [comment]
I see, Thank you.

**Comment by DeceptedPar** [comment]
your welcome, always welcome to helping people in need, if I know how to help that is, depending on the situation

**Comment by g3n-zu** [comment]
Understandable.

**Comment by DeceptedPar** [comment]
https://drive.google.com/file/d/1ocA9u78tEyjPJZXZlyBvZ9TzcHgKu0yQ/view?usp=sharing
I'm testing something out at the moment, I'm making Expurgation but I'm testing different voices

**Comment by g3n-zu** [comment]
I got it working! Updating All Chromatic Scales Now! Again, Thank you so much!

**Comment by DeceptedPar** [comment]
they're using the scales from this

**Comment by DeceptedPar** [comment]
your welcome!

**Comment by g3n-zu** [comment]
I do like how that sounds.

**Comment by DeceptedPar** [comment]
it's soft boyfriend and mike geno boyfriend

**Comment by DeceptedPar** [comment]
will have to put some of the parts an octave down though due to the scale not going as high as I need, that's not my problem though

**Comment by g3n-zu** [comment]
Right, Sorry about that. I tend to stop chromatics at Boyfriends highest note. Witch goes the highest out of all the Official chromatics.

**Comment by DeceptedPar** [comment]
yeah, I think you should go as high as you can go, people would love to see it!

**Comment by DeceptedPar** [comment]
I also wish senpai would go higher then he can, it kinda annoys me tbh

**Comment by g3n-zu** [comment]
Same! Senpai Chromatic is something else...

**Comment by DeceptedPar** [comment]
yeah

**Comment by DeceptedPar** [comment]
a scale of mike geno boyfriend would be good, I know you have one already, I mean like, one with just the normal bops, and the other one being extended

**Comment by g3n-zu** [comment]
What do you mean by normal?

**Comment by DeceptedPar** [comment]
basicaly the notes that aren't held if that helps, I'm not the best at explaining

**Comment by g3n-zu** [comment]
I see, I see. I'll see what I can do.

**Comment by DeceptedPar** [comment]
alright, if you can't do it, I'm not forcing you to do it

**Comment by DeceptedPar** [comment]
https://drive.google.com/file/d/1ocA9u78tEyjPJZXZlyBvZ9TzcHgKu0yQ/view?usp=sharing
updated it' got further, no longer mike geno, it's now fever

**Comment by g3n-zu** [comment]
I like how its sounding so far! Also I have an Expurgation Midi, If you would like it (Voices are Accurate and everything.)

**Comment by DeceptedPar** [comment]
ah, yes please that would help alot

**Comment by g3n-zu** [comment]
Check the Read me ; )

Expurgation Midi - https://drive.google.com/file/d/1p_JYSg6QGkzEBv3PWPNgS2IoSivtZBEo/view?usp=sharing

**Comment by DeceptedPar** [comment]
![image](https://user-images.githubusercontent.com/64243677/131178388-cc34f0b8-0613-4d89-a951-259cfaa45252.png)
been making this for a while, never gonna stop growing it, also thanks!

**Comment by g3n-zu** [comment]
Hehe : )

**Comment by DeceptedPar** [comment]
some notes are a bit off but I can fix them

**Comment by g3n-zu** [comment]
Oh heck my bad!

**Comment by DeceptedPar** [comment]
and it doesn't end right, but I can fix that aswell

**Comment by g3n-zu** [comment]
What do you mean? I'm sure it ends correctly...?

**Comment by DeceptedPar** [comment]
it ends early I mean

**Comment by DeceptedPar** [comment]
I fixed it though

**Comment by g3n-zu** [comment]
Oh...I exported it wrong....I forgot to extend it..my bad!

**Comment by DeceptedPar** [comment]
it's alright, I finished it though

**Comment by g3n-zu** [comment]
Nice! Can't wait to see the finished product!

**Comment by DeceptedPar** [comment]
also, this
![image](https://user-images.githubusercontent.com/64243677/131179788-8601ca93-7494-4853-b6dd-742f90e49f16.png)
is meant to look like this if I'm correct
![image](https://user-images.githubusercontent.com/64243677/131179854-3c844865-3716-4793-ba68-238b9889582b.png)

**Comment by g3n-zu** [comment]
Oh really? I didn't know that.

**Comment by DeceptedPar** [comment]
also bf notes are on color 2 so I can't hear them when I put soft boyfriends scale on it, hmm...

**Comment by g3n-zu** [comment]
You can do ALT C and change the color for all notes

**Comment by DeceptedPar** [comment]
https://drive.google.com/file/d/1ocA9u78tEyjPJZXZlyBvZ9TzcHgKu0yQ/view?usp=sharing
it's done

**Comment by DeceptedPar** [comment]
![image](https://user-images.githubusercontent.com/64243677/132056202-0397940b-5547-48bd-b55b-62b7aa311b96.png)
I realised this might help you out even more

**Comment by g3n-zu** [comment]
How might this help my lol, I’m at a lost..sorry

**Comment by DeceptedPar** [comment]
exporting the regions for samplers, actually I should've tested this out before telling you.

**Comment by g3n-zu** [comment]
I believe this is used for making soundfonts

**Comment by DeceptedPar** [comment]
ah

**Comment by DeceptedPar** [comment]
it exports each marker as a wav file OH.

**Comment by g3n-zu** [comment]
It sure dose lol

**Comment by DeceptedPar** [comment]
I'mma make your nene scale sing endless, not the normal endless, I recreated the voices and INSTRUMENTAL

**Comment by g3n-zu** [comment]
Oh dang, skills lol

**Comment by DeceptedPar** [comment]
should I make make nene sing against ron, or little man?

**Comment by g3n-zu** [comment]
Hmmmm…..Ron. I like his voice.

**Comment by DeceptedPar** [comment]
alrighty!!

**Comment by DeceptedPar** [comment]
![image](https://user-images.githubusercontent.com/64243677/132058219-6cd89fba-00c6-49b5-80df-e6f572152c6f.png)
I'mma upload it to google drive and link it here after

**Comment by DeceptedPar** [comment]
nene is gonna start the song, and then it'll be ron singing after her

**Comment by DeceptedPar** [comment]
if it's alright for me to link it here ofcourse

**Comment by g3n-zu** [comment]
Alright I’ll be lookin forward to it…

Also little update for I guess only you lol

1. Cablecrow Chromatic
2. Ace (Official) Chromatic 
3. Zardy Chromatic
4. Salty Chromatic 
5. Garcello (Extended) Chromatic.

And many more : )

**Comment by g3n-zu** [comment]
And yes you can send the link here.

: )

**Comment by DeceptedPar** [comment]
nice!!
I'm looking forward to them!

**Comment by DeceptedPar** [comment]
apparently garcello has an offical chromatic scale??
that's what I've heard, I dont know where to get it though

**Comment by g3n-zu** [comment]
Garcello sure dose have an official one but it seems to be private at the moment.

**Comment by DeceptedPar** [comment]
ah...

**Comment by DeceptedPar** [comment]
https://drive.google.com/file/d/1RKlk2_yGOB3qjqFFO-BW-EbQVLiQFCkk/view?usp=sharing
here it is!
tell me if you like it, it's my own custom vocals and instrumental

**Comment by g3n-zu** [comment]
Ummm..I’m on mobile currently Becuase I’m not home lol, do mind uploading it as mp3…?

**Comment by DeceptedPar** [comment]
ah, ok sorry! I'll get to that this instant

**Comment by DeceptedPar** [comment]
https://drive.google.com/file/d/1AUZLlaxg_5SWPqRSiLIDTvv1fzMY07pF/view?usp=sharing
mp3 version

**Comment by g3n-zu** [comment]
I’m so sorry for the late reply, but the file is saying I need access

**Comment by DeceptedPar** [comment]
ah.

**Comment by DeceptedPar** [comment]
https://drive.google.com/file/d/1AUZLlaxg_5SWPqRSiLIDTvv1fzMY07pF/view?usp=sharing
try now

**Comment by g3n-zu** [comment]
Alright, I’ll take a listen

**Comment by DeceptedPar** [comment]
I really like how my instrumental turned out

**Comment by g3n-zu** [comment]
I really like this.

**Comment by DeceptedPar** [comment]
![image](https://user-images.githubusercontent.com/64243677/132068790-991192f3-1a69-44cf-84a6-7c1058b1dce8.png)
Thanks!

**Comment by g3n-zu** [comment]
Dang such a pro lol

**Comment by DeceptedPar** [comment]
yeah, it was pretty easy to make due to it bein in a genesis soundfont

**Comment by g3n-zu** [comment]
Understandable. Also expect a Hex Chromatic soon too : )

**Comment by DeceptedPar** [comment]
and people think majins voice is just the trumpet from the genesis soundfont, actually it's 2 sounds
the trumpet and the clean guitar just layered ontop of each other
![image](https://user-images.githubusercontent.com/64243677/132068959-b07f7808-82b5-4910-be9e-b9a588e50553.png)

**Comment by DeceptedPar** [comment]
also now that I found the sample feature on slicex I might try to make a soundfont out of some of the scales, might add loop points if I can

**Comment by DeceptedPar** [comment]
and with the extended scales you've made I can for sure add loop points

**Comment by g3n-zu** [comment]
Oh dang. I though it was only 1 too lol

**Comment by DeceptedPar** [comment]
yeah, I realised when I was first making it, it sounded too clean, I added the clean guitar and it sounded really accurate

**Comment by DeceptedPar** [comment]
I'll send you a sample of the one I made when it's done exporting

**Comment by DeceptedPar** [comment]
https://drive.google.com/file/d/1HL-G4x50pJxYucINmkFtU1wU4_fjoBIg/view?usp=sharing
have a listen also boyfriends part is in it and I love how I changed it, I just raised it on octave
only did it for that one part

**Comment by g3n-zu** [comment]
Cool! Also I don’t know if I said this but I’m going to try and extend all my chromatic scales to have longer notes : )

**Comment by DeceptedPar** [comment]
nice!

**Comment by DeceptedPar** [comment]
also, the link I sent you is not the whole song, it's just part I sampled out

**Comment by g3n-zu** [comment]
I do like that, sounds nice.

**Comment by DeceptedPar** [comment]
also you can use utau samples and try to make the scales that way, just stretch out the samples and pitch them up
https://github.com/Blantados/blantados-funkin-utau-main
https://github.com/GenoX-Fome/funkin-utau
that is if you're up to the challenge

**Comment by g3n-zu** [comment]
Yeah, but sometimes it sounds weird, but I might. Who knows.

**Comment by DeceptedPar** [comment]
it's just a suggestion, I'm not forcing you to do it, it's up to you if you want to do it

**Comment by g3n-zu** [comment]
True true.

**Comment by DeceptedPar** [comment]
I'mma try to remaster endless with some different instruments

**Comment by g3n-zu** [comment]
Alright, have fun : )

**Comment by DeceptedPar** [comment]
alright, it's turning into an orchestral version now...

**Comment by DeceptedPar** [comment]
https://drive.google.com/file/d/1Uteyq3ad38nhvRe8BdP2w1Z6ye_KkcKg/view?usp=sharing
I finished it

**Comment by DeceptedPar** [comment]
it's slower in tempo so it's longer

**Comment by g3n-zu** [comment]
Holy fuck that’s a bop….

**Comment by DeceptedPar** [comment]
do you want the vocals and instrumental split apart?
you can have the instrumental seperate if you want

**Comment by DeceptedPar** [comment]
also it's 280 bpm

**Comment by DeceptedPar** [comment]
this time majins voice is a trumpet and a cello put together

**Comment by g3n-zu** [comment]
No thank you. But thanks for the offer

**Comment by DeceptedPar** [comment]
boyfriend is a romantic trumpet from the touhou soundfont

**Comment by g3n-zu** [comment]
I see,

**Comment by DeceptedPar** [comment]
yeah, I'll split the vocals and instrumental and send them you in a folder

**Comment by DeceptedPar** [comment]
and as you can see, I love music

**Comment by g3n-zu** [comment]
I love music too! In fact I made a remix of headache a while ago.

https://youtu.be/YDySkUSzegA

**Comment by DeceptedPar** [comment]
![image](https://user-images.githubusercontent.com/64243677/132074614-dac90d45-7d45-4924-947d-3f5899526130.png)
Woopsy daisy, look at what silly ol' me just did

**Comment by g3n-zu** [comment]
Oh sheet lol

**Comment by g3n-zu** [comment]
Thanks lol

**Comment by DeceptedPar** [comment]
you're welcome!

**Comment by DeceptedPar** [comment]
tbh, I wish I knew how to do utau easily, I wish there was there was a tool like FL studio that acts like utau cause you can't have 2 voices playing at the same time and you can't have multiple of the same samples from one voice singing at the same time

**Comment by g3n-zu** [comment]
I’m gonna assume this is how you found me (the video) but out of all these chromatics witch ones should be fixed? (https://youtu.be/a6sGfwwkHAk)

**Comment by g3n-zu** [comment]
And yeah, I wish there was a tool like that too. UTAU is very limited.

**Comment by DeceptedPar** [comment]
oh, I was looking on github by searching FNF and found this github

**Comment by g3n-zu** [comment]
That’s some pretty insane luck-

**Comment by DeceptedPar** [comment]
yeah, also that link, most of them seem to be either off-tune of some notes are wrong and some like updike sound a bit croaky

**Comment by g3n-zu** [comment]
Yeah. That’s why I plan to update all the chromatics : )

**Comment by DeceptedPar** [comment]
sarv, ruv and selever just don't sing appear when their image appear

**Comment by DeceptedPar** [comment]
wait no that;s just youtube acting up

**Comment by g3n-zu** [comment]
Lol YouTube is weird like that.

**Comment by g3n-zu** [comment]
Also I feel like I made some chromatics go to low. That’s why Updike and Whitty sound like that-

**Comment by DeceptedPar** [comment]
I might cover spotlight, a vocal cover, not a instrumental cover, I never do those, I only did endless cause it was easy to replicate

**Comment by g3n-zu** [comment]
True.

**Comment by DeceptedPar** [comment]
you should still make them go low like that, it might just be the sample though, but if you do make them go that low also make them go high so they'd be easy to make a song with a wide ranged of notes

**Comment by g3n-zu** [comment]
Also don’t tell anyone but my next video is most likely a Hope but everyone sings it cover (showcasing some new and updated chromatics)

**Comment by g3n-zu** [comment]
Yeah, it could just be the samples I chose lol

**Comment by DeceptedPar** [comment]
ooh... I'm looking forward to seing it

**Comment by DeceptedPar** [comment]
also I found the source code for the old monika build, the one with dreams of roses, it's such a good song, sad it had to go due to the drama

**Comment by g3n-zu** [comment]
Yeah, I honestly don’t understand why…like listening to that song…means you are like that person… I don’t know it’s all so weird.

**Comment by DeceptedPar** [comment]
>… like listening to that song…means you are like that person…

what do you mean by that?

**Comment by DeceptedPar** [comment]
https://drive.google.com/drive/folders/1_DBOdv1j5m0-XRyA4kflB8laH7MPY4Zr?usp=sharing
also here's the instrumental and vocals split into a folder

**Comment by g3n-zu** [comment]
Why was that song removed? Because the song maker was a [you know what]. The thing I don’t understand is why you can’t have the song in the mod?

**Comment by DeceptedPar** [comment]
ah, makes sense now, it's stupid yeah....

**Comment by g3n-zu** [comment]
Yeah.

also this is a cool thing I did lol - https://youtu.be/vvtT_yRdsrk

**Comment by DeceptedPar** [comment]
I like it!

**Comment by DeceptedPar** [comment]
https://github.com/mamontime/fnf-utau-voicebanks
https://github.com/LivyNight/livynight-funkin-utau
found two new utau githubs

**Comment by kubapl2004gmail** [comment]
Tari from smg4 mod by cubethepenguin chromatic scale pls?
Belle frontiere from smg4 mod by cubethepenguin chromatic scale PLS?

**Comment by g3n-zu** [comment]
I will make those chromatics sooner or later.

**Comment by kubapl2004gmail** [comment]
Soft GF chromatic scale PLS?

**Comment by kubapl2004gmail** [comment]
Connor SSN chromatic scale PLS?

**Comment by kubapl2004gmail** [comment]
Mom.exe SSN chromatic scale PLS?

**Comment by kubapl2004gmail** [comment]
K.o.u chromatic scale PLS?

**Comment by g3n-zu** [comment]
Can you please not spam here.

**Comment by g3n-zu** [comment]
And as I said, I will make the chromatics sooner or later

**Comment by kubapl2004gmail** [comment]
Hank chromatic scale PLS?

**Comment by kubapl2004gmail** [comment]
Jebus chromatic scale PLS?

**Comment by kubapl2004gmail** [comment]
fix cj chromatic scale to fit the pitch of key notes pls?

**Comment by DeceptedPar** [comment]
if the whole scale is up by one key, it doesn't matter.
if random notes are up by one or more key then that should be fixed but from what you're saying, the scale is up by one, it doesn't matter, and is not going to be fixed, I dont make these but I know it doesn't matter enough to be fixed

**Comment by g3n-zu** [comment]
Are you talking to me? (I’m a bit confused)

**Comment by DeceptedPar** [comment]
I was talking to kubapl

**Comment by g3n-zu** [comment]
I see, I see.

**Comment by DeceptedPar** [comment]
I don't know why they follow me, they just do and that's how they found this

**Comment by g3n-zu** [comment]
It’s alright, they massage me on discord all the time…It gets annoying sometimes

**Comment by DeceptedPar** [comment]
not to sound rude, never talked to them and they already sound like the annoying kind of person
